modules/opensearch/includes/opensearch.search.inc

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * @file
+ * Implements handling of loans through OpenRuth.
+ */
+
+/**
+ * Get a list of material types from the Well.
+ */
+function opensearch_search_material_types() {
```

Comment:
It would also conflict with my current mode of development where I have connie enabled. But it would be nice with a reference dummy implementations. 

One way to go about it: I could implement it in a seperate connie_search module?